src/style.css

```diff
@@ -0,0 +1,51 @@
+@font-face {
+  font-family: "Avenir";
+  src: url("./Avenir/Avenir-Book.ttf");
+  font-weight: 300;
+}
+
+.form-for-searching {
+  margin-top: 20px;
+}
+
+.searc_bar {
+  width: 100%;
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+  font-family: "Avenir", serif;
+  background: url(images/Search.svg) no-repeat scroll 26px 28px;
+  color: #3d4e61;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+}
+
+.searc_bar_big {
+  font-size: 16px;
+  line-height: 22px;
+  background-position: top 25px left 26px;
+  padding-left: 62px;
+
+  height: 70px;
+}
+
+.searc_bar_small {
+  padding-left: 33px;
+  height: 42px;
+  background: url(images/Search.svg) no-repeat scroll 15px 15px;
+  background-size: 11px;
+  background-position: top 14px left 12px;
+  font-size: 14px;
+  line-height: 19px;
+}
```

Comment:
We don't have any basic search bar, so why we should use there big and small search class names? We have search-input and search-input - small (--small). Therefore, we set all styles that are supposed to be for the "large" input for all inputs in general, and with the modifier small we change only the necessary values to make our input smaller.

Reply:
Thank you for a good advice!